src/Jackett.Common/Definitions/nekobt.yml

```diff
@@ -0,0 +1,250 @@
+---
+id: nekobt
+name: nekoBT
+description: "nekoBT is a Private Anime Torrent Tracker specializing in anime content with subtitle and fansub filtering"
+language: en-US
+type: public
+encoding: UTF-8
+links:
+  - https://nekobt.to/
+
+caps:
+  categorymappings:
+    - {id: 5000, cat: TV, desc: "TV"}
+    - {id: 5070, cat: TV/Anime, desc: "Anime"}
+    - {id: 2000, cat: Movies, desc: "Movies"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, tvdbid]
+    movie-search: [q]
+
+settings:
+  - name: apikey
+    type: text
+    label: API Key (Optional)
+  - name: info_key
+    type: info
+    label: About your API key
+    default: "API key is optional. You can find or generate a new API Key by accessing your <a href=\"https://nekobt.to/\" target=\"_blank\">nekoBT</a> account settings page if needed."
+  - name: info_filters
+    type: info
+    label: Advanced Filters
+    default: "Configure optional filters below. Leave empty to disable filtering."
+  - name: levels
```

Comment:
Most of these config settings can be removed. I think the only ones worth keeping are audio and sub language, and maybe group ID.